### PR TITLE
Nbt type ids unpicked

### PR DESCRIPTION
**Pull request by Earthcomputer**

### CODE REVIEW

File: mappings/net/minecraft/nbt/NbtElement.mapping [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-	COMMENT <li>{@code 7} - byte array</li>
-	COMMENT <li>{@code 8} - string</li>
-	COMMENT <li>{@code 9} - list</li>
-	COMMENT <li>{@code 10} - compound object</li>
-	COMMENT <li>{@code 11} - int array</li>
-	COMMENT <li>{@code 12} - long array</li>
-	COMMENT </ul>
+	COMMENT Each type of NBT element has an associated integer {@linkplain NbtElement#getType NBT type}.
+	COMMENT
+	COMMENT @see net.fabricmc.yarn.constants.NbtTypeIds for a list of valid types.
```
Comment (liach) [inline mappings/net/minecraft/nbt/NbtElement.mapping]: Consider add this `@see` to the doc of the `getType` method as well.

File: src/constants/java/net/fabricmc/yarn/constants/NbtTypeIds.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	public static final int INT = 3;
+	public static final int LONG = 4;
+	public static final int FLOAT = 5;
+	public static final int DOUBLE = 6;
+	public static final int BYTE_ARRAY = 7;
+	public static final int STRING = 8;
+	public static final int LIST = 9;
+	public static final int COMPOUND = 10;
+	public static final int INT_ARRAY = 11;
+	public static final int LONG_ARRAY = 12;
```
Comment (liach) [inline src/constants/java/net/fabricmc/yarn/constants/NbtTypeIds.java:102]: I recommend some minimal doc that at least link to their respective classes.

File: mappings/net/minecraft/nbt/NbtElement.mapping [hunk 3]
```diff
@@ -12,6 +8,8 @@ CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
 		COMMENT Gets the type of this NBT element.
 		COMMENT
 		COMMENT @return the type
+		COMMENT
+		COMMENT @see net.fabricmc.yarn.constants.NbtTypeIds for a list of valid types.
```
Comment (liach) [inline mappings/net/minecraft/nbt/NbtElement.mapping]: Fyi this renders something like

`for a list of valid types.` as the description for the link to `net.fabricmc.yarn.constants.NbtTypeIds`. You know, javadoc is weird.
```suggestion
		COMMENT @see net.fabricmc.yarn.constants.NbtTypeIds a list of valid types
```

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Comment by liach** [comment]
If you don't mind, consider pulling in the latest 11a and add a cc0 license header to the constant java file.

**Review by modmuss50 — APPROVED** [review]
👍 I can merge + fix headers if needed.

**Comment by liach** [comment]
Then I recommend merging right now

**Comment by liach** [comment]
Hmm, maybe you can merge a few unpick prs together and run one round of updateLicenses.

**Pull request merged**